app/styles/base.css

```diff
@@ -30,30 +30,30 @@
  */
 
 :root {
-  font-family: sans-serif;
+    font-family: sans-serif;
```

Comment:
I guess this unfortunately breaks `git blame` for basically every line in these files? :/